jrnl/plugins/datecount_exporter.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# encoding: utf-8
+# Copyright (C) 2012-2021 jrnl contributors
+# License: https://www.gnu.org/licenses/gpl-3.0.html
+from collections import Counter
+
+from .text_exporter import TextExporter
+
+
+class DatecountExporter(TextExporter):
+    """This Exporter lists dates and their respective counts, for heatingmapping etc."""
+
+    names = ["datecount"]
+    extension = "datecount"
+
+    @classmethod
+    def export_entry(cls, entry):
+        raise NotImplementedError
+
+    @classmethod
+    def export_journal(cls, journal):
+        """Returns dates and their frequencies for an entire journal."""
+        date_counts = Counter()
+        for entry in journal.entries:
+            # entry.date.date() gets date without time
+            date = str(entry.date.date())
+            date_counts[date] += 1
+        result = "\n".join(f"{date}, {count}" for date, count in date_counts.items())
```

Comment:
Can we make this format match the output of `--tags` (using colons)? Termgraph support custom delimiters anyway, so it won't break anything with integration, and gives our output a more uniform look.

Reply:
Also, maybe we should respect the date format in the config file? It would give users more flexibility with how they use the plugin.

Reply:
I can make it match the output of `--tags`, but I am not sure about respecting the config's date format. In #743, you call for this plugin to maintain output consistency. Have I misunderstood something?

Reply:
That a good point. I was thinking that since this command doesn't need to be piped through intermediate commands anymore that the added configurability could be useful. But I think you're right that it's best to stick to one format for now.

Reply:
Thank you for clarifying 👍

Reply:
> Can we make this format match the output of --tags (using colons)?

Hmm... I am not sure how standard colons are. But I can definitely make the change if consistency's necessary.

Reply:
I think `--tags` uses colons because commas are already present to separate tags. Wouldn't that make it a special case? Again, I feel like using commas is more standard (at-least `Termgraph` accepts that by default).